### DISCUSSION TITLE
Why are activity icons hidden after 5s?

### DISCUSSION THREAD

**Discussion by klaernie** [sni/thruk#1275] [Q&A] [answered]
I've been wondering for a while, why the spinner icons / activity icons disappear after 5 seconds.

In my mind the view shown to the user represents the state as it was when it was generated. So it's a bit puzzling, that the "check is executing" indication gets hidden after 5 seconds, while the check might actually still be running.

In my mind the spinners should remain visible until the next refresh of the page.
Am I missing a viewpoint for hiding the spinners?

**Comment by sni** [comment, marked as answer]
The thing is, Thruk does not know when exactly a check is finished until it refreshes the hole page. So i wanted to reduce the noise of the page and not have lots of animated things visible all the time but still have a initial indicator.

**Reply by klaernie** [reply]
Cool. Thanks a lot for explaining. Would you consider making this configurable? I'd probably tune it to 120s, since if I'm actively waiting for a check to complete I'm going to refresh every 10s anyway, but for keeping the page open in the background or on another display it could indeed be distracting some time later.

**Comment by sni** [comment]
there you go: 52709367eb1b809c1181c35c4e49c2b8e0669b91

**Reply by klaernie** [reply]
Thank you very much!
